Sources/FluidAudio/TextToSpeech/Kokoro/Pipeline/InSynth/KokoroSynthesizer.swift

```diff
@@ -0,0 +1,750 @@
+import Accelerate
+import CoreML
+import Foundation
+import OSLog
+
+#if canImport(FoundationNetworking)
```

Comment:
nit on the folder name, maybe just have it as "synthesize" and "preorpocess" and "postprocess"? snyth is a bit unclear